lldb/test/API/tools/lldb-dap/stepInTargets/TestDAP_stepInTargets.py

```diff
@@ -10,9 +10,10 @@
 
 
 class TestDAP_stepInTargets(lldbdap_testcase.DAPTestCaseBase):
-    @skipIf(
-        archs=no_match(["x86_64"])
-    )  # InstructionControlFlowKind for ARM is not supported yet.
+    @skipIf
+    # InstructionControlFlowKind for ARM is not supported yet.
+    # On x86_64 lldb-dap seems to ignore targetId when stepping into functions.
```

Comment:
Judging by everything said on this thread, I believe this is only true on x86_64 *windows*, so we should limit the exclusion to windows. Then you may actually be able to keep the xfail decorator, since the test is really expected to fail in this configuration (until the issue is fixed). (i.e., I think this should use `@expectedFailureAll(oslist=["windows"])` instead of the `@skipIf`).

Reply:
Ah thanks for catching that. I added back the original `@skipIf` since the comment indicates that running the test on ARM is a waste, given that InstructionControlFlowKind is unsupported, and combined that with the `@expectedFailureAll` you gave above to expect failure on Windows configurations.

Reply:
Great, that's exactly what I had in mind. Thanks.